app/components/Form/ProjectFormSummary.tsx

```diff
@@ -4,112 +4,133 @@ import readOnlyTheme from "lib/theme/ReadOnlyTheme";
+              year
+              fundingStreamByFundingStreamId {
+                description
               }
             }
+            projectStatusByProjectStatusId {
+              name
+            }
           }
-          allOperators {
-            edges {
-              node {
-                rowId
+          formChangeByPreviousFormChangeId {
+            newFormData
+            asProject {
+              operatorByOperatorId {
                 legalName
                 bcRegistryId
               }
-            }
-          }
-          allProjectStatuses {
-            edges {
-              node {
+              fundingStreamRfpByFundingStreamRfpId {
+                year
+                fundingStreamByFundingStreamId {
+                  description
+                }
+              }
+              projectStatusByProjectStatusId {
                 name
-                rowId
               }
             }
           }
         }
-      `,
-      props.query
-    );
+      }
+    `,
+    props.projectRevision
+  );
 
-  const isOverviewEmpty = useMemo(() => {
-    return Object.keys(projectFormChange.newFormData).length === 0;
-  }, [projectFormChange.newFormData]);
+  const newDataAsProject = projectFormChange.asProject;
+  const previousDataAsProject =
+    projectFormChange.formChangeByPreviousFormChangeId?.asProject;
 
-  const selectedOperator = useMemo(() => {
-    return allOperators.edges.find(
-      ({ node }) => node.rowId === projectFormChange.newFormData.operatorId
-    );
-  }, [allOperators, projectFormChange.newFormData.operatorId]);
+  const oldUiSchema = previousDataAsProject
+    ? createProjectUiSchema(
+        previousDataAsProject.operatorByOperatorId.legalName,
+        previousDataAsProject.operatorByOperatorId.bcRegistryId,
+        `${previousDataAsProject?.fundingStreamRfpByFundingStreamRfpId?.fundingStreamByFundingStreamId.description} - ${previousDataAsProject?.fundingStreamRfpByFundingStreamRfpId?.year}`,
+        previousDataAsProject.projectStatusByProjectStatusId.name
+      )
+    : null;
 
-  const rfpStream = useMemo(() => {
-    return allFundingStreamRfps.edges.find(
-      ({ node }) =>
-        node.rowId === projectFormChange.newFormData.fundingStreamRfpId
-    );
-  }, [allFundingStreamRfps, projectFormChange.newFormData.fundingStreamRfpId]);
+  // Filter out fields from the formData that have not changed from the previous revision so the summary ignores these fields
+  const filteredSchema = JSON.parse(JSON.stringify(projectSchema));
+  const filteredFormData = useMemo(() => {
+    const newDataObject = {};
+    for (const [key, value] of Object.entries(projectFormChange.newFormData)) {
+      if (
+        value ===
+        projectFormChange.formChangeByPreviousFormChangeId?.newFormData?.[key]
+      )
+        delete filteredSchema.properties[key];
+      else newDataObject[key] = value;
+    }
+    return newDataObject;
+  }, [
```

Comment:
I don't think we need to keep in memory a copy of the original schema/formData and a copy of the new one. we can, depending on the `isFirstRevision` property, generate the data to use:
```tsx
const [formSchema, formData]  = useMemo( () => {
  if(isFirstRevision) return [projectSchema, projectFormChange.newFormData];

  const filteredSchema = JSON.parse(JSON.stringify(projectSchema));
  ...etc...
  return [filteredSchema, newDataObject]
}
```

This way we don't have logic in the JSX further down